server/routes/api/invitation/accept.get.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+import { getInvitationByToken } from "~/server/services/invitation";
+import { addCollaboratorById } from "~/server/services/workspace";
+
+const schema = z.object({
+  token: z.string(),
+});
+
+export default defineEventHandler(async (event) => {
+  const { user } = await requireUserSession(event);
```

Comment:
We should redirect the user to the login page if they are not logged in. We can add a redirect link to the sign in page so they make their way back here automatically.